vms/registration/templates/registration/signup_volunteer.html

```diff
@@ -214,7 +214,18 @@
                     {% trans "City" %}<span class="asteriskField">*</span>
                 </label>
                 <div class="controls">
+<!--
```

Comment:
You can simply remove this. Git is there, every line we can figure out later if we need. :)